### PR TITLE
feat(front) : [Comment] add chatbox layout

### LINKED ISSUE #2923: [Comment] overflow management and customization
Longs words can trigger overflow :
image

It should also be great to add more customization to the comment component :

- White / product 100 background for messages
- A chatbot layout

Ex (Poplee Engagement Hotline)
image

### PR DESCRIPTION
**Pull request by BertrandPodevin**
## Description

Add chatbox layout for the Comment component

image

[Figma](https://www.figma.com/design/CVCTTj29ofwdSwIyHE9CnS/Ligne-directe-(PEng)?node-id=701-80&t=nNXPzA4tju0lf0sS-0)
image

closes #2923 

-----

Optionally, technical or more in-depth description for reviewers.
Keep an empty line under your text, as well as the 5 lines that follow it.

-----

### CODE REVIEW

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/19482/ui-diff)

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2932/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/19483/ui-diff)

File: packages/scss/src/components/comment/component.scss [hunk 1]
```diff
@@ -43,7 +43,7 @@
 		}
 
 		.comment-content {
-			background: var(--palettes-neutral-50);
+			background: var(--components-comment-background-color, var(--palettes-neutral-50));
```
Comment (vvalentin-lucca) [inline packages/scss/src/components/comment/component.scss:46]:
```suggestion
			background-color: var(--components-comment-background-color, var(--palettes-neutral-50));
```

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2932/storybook

**Comment by jeremie-lucca** [comment]
Implemented here: https://github.com/LuccaSA/lucca-front/pull/2866

**Pull request closed without merging**